app/models/SubmissionProcessingResult.scala

```diff
@@ -16,16 +16,34 @@
 
 package models
 
+import play.api.mvc.Result
+import play.api.mvc.Results.BadRequest
+import play.api.mvc.Results.InternalServerError
+import play.api.http.Status.BAD_REQUEST
+import play.api.http.Status.FORBIDDEN
+
 sealed trait SubmissionProcessingResult
 
 object SubmissionProcessingResult {
 
   case object SubmissionSuccess extends SubmissionProcessingResult
 
-  sealed trait SubmissionFailure        extends SubmissionProcessingResult
+  sealed trait SubmissionFailure extends SubmissionProcessingResult
+
   case object SubmissionFailureInternal extends SubmissionFailure
+
   case object SubmissionFailureExternal extends SubmissionFailure
-  case object SubmissionFailureRejected extends SubmissionFailure
+
+  case class SubmissionFailureRejected(httpStatus: Int, asString: String) extends SubmissionFailure {
+
+    def recodeForUpstream(): Result =
+      httpStatus match {
+        case BAD_REQUEST =>
+          BadRequest(asString)
+        case FORBIDDEN =>
+          InternalServerError
+      }
+  }
```

Comment:
There is no need for the recode or expose these details outside of the service layer or need . The Forbidden shouldn't be mapped to this at all, this indicates an error in logic somewhere else